lib/Settings/Admin/AdminSettings.php

```diff
@@ -502,24 +502,32 @@ protected function getGroupDetailsArray(array $gids, string $configKey): array {
 		return $groups;
 	}
 
-	protected function validApachePHPConfiguration(): bool {
+	protected function validApachePHPConfiguration(): string {
+		if (!function_exists('exec')) {
+			return 'unknown';
+		}
+
 		$output = [];
-		@exec('apachectl -M | grep mpm', $output, $returnCode);
+		try {
+			@exec('apachectl -M | grep mpm', $output, $returnCode);
```

Comment:
"apachectl -M" don't work on Fedora (and perhaps on all RedHat derivated distros)
You have instead to use "httpd -M"